src/test/groovy/test/TestUtils.groovy

```diff
@@ -19,16 +23,29 @@ final class TestUtils {
       .withPluginClasspath()
       .build()
   }
-
-  static def getLicenseText(def fileName) {
-    getClass().getResource("/license/${fileName}").text
+  static def myGetLicenseText(String fileName) {
+    return getLicenseText(fileName)
   }
 
   static def assertJson(def expected, def actual) {
     return parser.parse(actual) == parser.parse(expected)
   }
 
+  static def htmlToXml(String text) {
+    // Convert HTML into legal-enough XML that we can use the XML comparison
+    // utility to compare two HTML strings. This is only just what we need for
+    // this exact case, so update as needed.
+    text = text.replaceAll('<br>', '<br/>')
+    text = text.replaceAll('<hr>', '<hr/>')
+    text = text.replaceAll('&copy;', '(c)')
```

Comment:
Sorry, one last thing before we merge, why remove copy with `(c)`?

Reply:
The XML comparison recognizes that '&copy;' is an "entity" (ampersand-semicolon), but doesn't have the copyright entity built-in, yielding a complaint about "undefined entity" and failing the test.  The replacement doesn't really matter, but that seemed obvious.  (The message the XML comparison generates is obvious only after you've figured out what it means!)